source/js/kitchen.js

```diff
@@ -146,24 +162,35 @@ function loadTemplateFromURL(templateURL, callback) {
     });
 }
 
+function presenterForType(type) {
+    console.log(type);
```

Comment:
Yeah sure! Sorry, meant to get rid anyway but it will print to the web console. You can see it by opening the inspector on the simulator from Safari.